.serena/memories/project_structure.md

```
@@ -0,0 +1,68 @@
+# Project structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)